bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/OnboardingScreen.kt

```diff
@@ -0,0 +1,20 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+
+class OnboardingScreen {
+
+    fun assertSkipButton(): OnboardingScreen {
```

Comment:
[Suggestion] Since the test checks the text displayed on the view, could the title of the method be updated to something like `assertSkipButtonText` ?

Reply:
Sure, I can.